### PR TITLE
Do Not Merge - Blocked on Release - 46047 Changing Community License

### PR DESCRIPTION
**Pull request by paigecalvert**
SC: https://app.shortcut.com/replicated/story/46047/orange-team-support-changing-a-license

Preview: https://deploy-preview-278--replicated-docs.netlify.app/enterprise/updating-licenses#changing-a-community-license

Related vendor content added in this PR: https://github.com/replicatedhq/replicated-docs/pull/279

### CODE REVIEW

**Comment by ajp-io** [comment]
This shows the modal and the button in the background. Just FYI. 
image

File: docs/enterprise/updating-licenses.md [hunk 1]
```diff
@@ -23,3 +23,12 @@ The admin console will then check if there are any updates on your current licen
...
 - A "License uploaded successfully" note will appear when the changes are successfully applied.
+
+## Changing a Community License
+
+You can change a community license to a paid license. This allows you to upgrade to a paid license without having to reinstall the application.
+
+To change a community license to a paid license:
+
+1. In the License tab of the admin console, click **Change license**.
+1. In the dialog, upload the new, paid license file that you received from your vendor.
```
Comment (ajp-io) [inline docs/enterprise/updating-licenses.md:34]:
```suggestion
1. In the dialog, upload the new license file that you received from your vendor.
```

**Review by ajp-io — CHANGES_REQUESTED** [review]

**Comment by paigecalvert** [comment]
Thanks @ajp-io, great corrections. Q: is it important at all to differentiate between changing to a new license, and changing to a new _type_ of license? I know for non-community licenses, users would click the sync button to update. For air gap, I suppose you technically do upload a "new" license. Anyway, not sure if this is important to differentiate or not, wanted to throw it out there.

**Comment by ajp-io** [comment]
@paigecalvert That is a good question. It's definitely an interesting distinction. In a sense, you're not uploading a new license in air gap. It's the same license as far as the vendor portal is concerned, but you upload a different license _file_, since perhaps the details of the license have changed.

I view "new/different license" as a different customer (i.e., license) created in the vendor portal. And in that sense, we are changing the license here since it's a new customer (no longer using the one community customer/license that all community users upload). Whereas changing the license type can be accomplished with the same license—you just have to sync it.

So I'm okay with just saying that you're changing the license, since from a vendor portal POV that means you have created a new customer—you haven't just changed settings for an existing customer.

**Review by ajp-io — APPROVED** [review]

**Pull request merged**